app/src/main/res/layout/fragment_data.xml

```diff
@@ -1,132 +1,248 @@
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/fragmentDataUploaded"
+                android:layout_width="0dp"
+                android:layout_height="@dimen/height_forty"
+                android:layout_marginTop="16dp"
+                android:layout_weight="3"
+                android:textColor="@color/black"
+                android:textIsSelectable="true"
+                android:textSize="@dimen/text_size_seventeen"
+                app:layout_constraintStart_toStartOf="@+id/tree_data_guideline"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/fragmentDataWaitingUploadLabel"
+                android:layout_width="0dp"
+                android:layout_height="@dimen/height_forty"
+                android:layout_marginTop="16dp"
+                android:layout_weight="3"
+                android:text="@string/fragment_data_waiting_to_upload_label"
+                android:textColor="@color/black"
+                android:textSize="@dimen/text_size_seventeen"
+                app:layout_constraintStart_toStartOf="@+id/fragmentDataUploadedLabel"
+                app:layout_constraintTop_toBottomOf="@+id/fragmentDataUploadedLabel" />
+
+            <TextView
+                android:id="@+id/fragmentDataWaitingUpload"
+                android:layout_width="0dp"
+                android:layout_height="@dimen/height_forty"
+                android:layout_marginTop="16dp"
+                android:layout_weight="3"
+                android:textColor="@color/black"
+                android:textIsSelectable="true"
+                android:textSize="@dimen/text_size_seventeen"
+                app:layout_constraintStart_toStartOf="@+id/fragmentDataUploaded"
+                app:layout_constraintTop_toBottomOf="@+id/fragmentDataUploaded" />
+
+            <TextView
+                android:id="@+id/fragmentDataValidatedTreesLabel"
+                android:layout_width="0dp"
+                android:layout_height="@dimen/height_forty"
+                android:layout_marginTop="16dp"
+                android:layout_weight="3"
+                android:text="@string/fragment_data_validated_label"
+                android:textColor="@color/black"
+                android:textSize="@dimen/text_size_seventeen"
+                app:layout_constraintStart_toStartOf="@+id/fragmentDataWaitingUploadLabel"
+                app:layout_constraintTop_toBottomOf="@+id/fragmentDataWaitingUploadLabel" />
+
+            <TextView
+                android:id="@+id/fragmentDataValidatedTrees"
+                android:layout_width="0dp"
+                android:layout_height="@dimen/height_forty"
+                android:layout_marginTop="16dp"
+                android:layout_weight="3"
+                android:textColor="@color/black"
+                android:textIsSelectable="true"
```

Comment:
A lot of the text in this view is selectable, why is that?

Reply:
Not sure how this got in, must be my IDE doing it. Removing all of these from this layout